hdf/util/hdfpack.c

```diff
@@ -444,6 +444,7 @@ copy_blocks(mydd_t *dd, int32 infile, int32 outfile)
         }
     }
     Hendaccess(outaid);
+    Hendaccess(inaid);
```

Comment:
Did you notice some of the Hendaccess statements have if (ret == FAIL) but some don't?